src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineRewriter.vb

```diff
@@ -375,6 +375,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Sub
 
+        Friend Function EnsureSpecialMember(member As SpecialMember, bag As DiagnosticBag) As Symbol
```

Comment:
Consider adding similar overload for EnsureSpecialType to be consistent and rely on errors in the bag for all checks.